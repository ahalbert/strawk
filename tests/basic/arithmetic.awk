BEGIN {
  print 1+2, 2-3, 4*5, 6/7, 8%9, 10^11
  print "----"
  a = 1
  a += 1
  print a
  a -= 1
  print a
  print ++a
  print a++
  print a
  a *= 2
  print a
  a /= 2
  print a
  a %= 3
  print a
  a ^= 2
  print a
}
