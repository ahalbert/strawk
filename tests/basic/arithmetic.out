3 -1 20 0.85714 8 100000000000
----
2
3
4
4
5
10
5
2
4
